src/sentry/static/sentry/app/components/commitLink.jsx

```diff
@@ -2,67 +2,68 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import {t} from 'app/locale';
+import Button from 'app/components/buttons/button';
 import InlineSvg from 'app/components/inlineSvg';
 
-const isBitbucket = providerId =>
-  ['bitbucket', 'integrations:bitbucket'].includes(providerId);
-const isGithub = providerId =>
-  ['github', 'integrations:github', 'integrations:github_enterprise'].includes(
-    providerId
-  );
+// TODO(epurkhiser, jess): This should be moved into plugins.
+const SUPPORTED_PROVIDERS = [
+  {
+    icon: 'icon-github',
+    providerIds: ['github', 'integrations:github', 'integrations:github_enterprise'],
+    commitUrl: v => `${v.baseUrl}/commit/${v.commitId}`,
```

Comment:
Let's destructure or use a better param name

Reply:
I used `v` here since I was trying to keep the line short, and since the single line function is pretty self explanatory. I can switch it over to destructuring though.

Reply:
I switched it over to desctructuring, but I'm curious to hear more of your thoughts around it vs what I was doing before